spec/license_scout/dependency_manager/cpan_spec.rb

```diff
@@ -290,19 +290,70 @@ def fixture_pkg(pkg_name)
         expect(capture_tiny.license_files.first).to eq(expected_path)
 
         libintl = cpan.dependencies.find { |d| d.name == "libintl-perl" }
-        expect(libintl.license).to be_nil
+        expect(libintl.license).to eq("GPL-3.0") # from default overrides
```

Comment:
seems like we should still have one test case where the license and license file are not found and there isn't an override.